src/app/workspace/organization/users/organization-users.component.html

```diff
@@ -1 +1,151 @@
-Users tab
+<div class="container" *ngIf="(organization$ | async) as organization">
+  <div class="d-flex">
+    <h5 class="text-muted flex-grow-2"
+        i18n="@@organizations.users.header">
+      Users in the <i class="{{ organization.icon }} mr-1"></i>{{ organization.name }} organization
+    </h5>
+
+    <i class="fas fa-plus text-success large-text clickable"
+       aria-hidden="true"
+       title="Add User"
+       i18n-title="@@organizations.users.add"
+       (click)="addUser()">
+    </i>
+  </div>
+
+  <ng-container *ngIf="(users$ | async)?.length else emptyResult">
+
+    <div class="form-control d-flex p-0 m-2 filter">
+      <input class="border-0 flex-grow-2 px-2" placeholder="Filter users"/>
+
+      <button class="btn search-button"
+              type="button"
+              title="Filter"
+              i18n-title="@@organizations.users.filter">
+        <i class="fa fa-search" aria-hidden="true"></i>
+        <span class="d-none" i18n="@@organizations.users.filter.button">Filter</span>
+      </button>
+    </div>
+
+    <table>
+      <thead class="thead-light medium-text">
+      <tr>
+        <td colspan="3">
+        </td>
+        <td class="px-2"
+            i18n="@@organizations.user.roles.create">
+          Create Project
+        </td>
+        <td class="px-2"
+            i18n="@@organizations.user.roles.manage">
+          Manage Organization
+        </td>
+        <td colspan="3">
+        </td>
+      </tr>
+      </thead>
+      <tbody>
+      <ng-container *ngFor="let user of users$ | async">
+        <tr>
+          <td class="px-1">
+            <i class="fa fa-caret-right large-text text-success {{ expanded[user.email] ? 'fa-rotate-90' : null }}"
+               aria-hidden="true"
+               title="{{ expanded[user.email] ? 'Close' : 'Open' }}"
+               i18n-title="@@organizations.user.expand"
+               (click)="(expanded[user.email] = !expanded[user.email])"></i>
+          </td>
+
+          <td class="px-1">
+            <input class="form-control edit-border"
+                   type="text"
+                   tabindex="1"
+                   placeholder="Username"
+                   [(ngModel)]="user.name"
+                   (blur)="updateUserName(user)">
+          </td>
+
+          <td class="px-1">
+            <input class="form-control edit-border"
+                   type="email"
```

Comment:
I am not sure you should be able to modify the email since it is managed by Keycloak.

Reply:
Ok. Will it be changeable by Keycloak?